src/plugins/intel_gpu/src/graph/include/kernel_selector_helper.h

```diff
@@ -4,23 +4,22 @@
 
 #pragma once
```

Comment:
looks this file contains only format changes without any new logic? If yes better to simple revert it

Reply:
Done